dds/DCPS/XTypes/TypeLookupService.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
```

Comment:
By convention we wrap pragma-onces in a ACE check.

Reply:
https://github.com/objectcomputing/OpenDDS/blob/3354da116616da4f05476a964ae97c83a191bad5/dds/DCPS/Serializer.h#L43-L45

If you need an example.

Reply:
Yet again nevermind.